tests/copy.bats

```diff
@@ -21,6 +21,13 @@ load helpers
   buildah copy $cid ${TESTDIR}/randomfile
   buildah copy $cid ${TESTDIR}/other-randomfile ${TESTDIR}/third-randomfile ${TESTDIR}/randomfile /etc
   buildah rm $cid
+
+  cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json alpine)
+  root=$(buildah mount $cid)
+  buildah config --workingdir / $cid
+  buildah copy $cid "${TESTDIR}/*randomfile" /etc
+  (cd ${TESTDIR}; for i in *randomfile; do stat ${root}/etc/$i; done)
```

Comment:
Please use `cmp` here.  (Aside: `stat` could be replaced here by `test -s`, which we use elsewhere, but then `cmp` is going to flag an error if the file doesn't exist, too.)